documentation/decisions/adr/2023-12-18-measurement-dashboard-architecture.md

```diff
@@ -100,25 +158,31 @@ This option involves building a custom dashboard app using an existing framework
 This option involves selecting and hosting an open source dashboard solution, such as [Metabase][metabase] or [Redash][redash]. This solution would most likely be self-hosted and connect directly to our data warehouse, and enable business analysts to build adhoc reporting and dashboards with SQL and a drag-and-drop interface. Individual dashboards can then be configured for broader publication to external stakeholders.
 
 > [!TIP]
-> **Bottom line:** This option would be best if we want to adopt an open source solution that enables business analysts to build and host dashboards with minimal support from engineers, but are willing to compromise on the amount of control we have over the look and feel of those dashboards and don't need to expose analytics data via API.
+> **Bottom line:** This option would be best if:
+> - we want to adopt an open source solution that enables business analysts to build *and* host dashboards with minimal support from engineers,
+> - but we are willing to commit the upfront resources needed to set up the infrastructure *and* can compromise on the amount of control we have over the look and feel of those dashboards.
```

Comment:
I love the new if / but framework for tips! Nice innovation.

Reply:
ooh this is awesome